stetho/src/main/java/com/facebook/stetho/common/android/FragmentActivityAccessor.java

```diff
@@ -6,10 +6,9 @@
 
 import javax.annotation.Nullable;
 
-public interface FragmentActivityAccessor {
+public interface FragmentActivityAccessor<
+    FRAGMENT_ACTIVITY extends Activity,
+    FRAGMENT_MANAGER> {
   @Nullable
-  public Object getFragmentManager(Activity fragmentActivity);
-
-  @Nullable
-  public Object getSupportFragmentManager(Activity fragmentActivity);
+  public FRAGMENT_MANAGER getFragmentManager(FRAGMENT_ACTIVITY activity);
```

Comment:
Okay. I wanted to make sure this had been taken into consideration. This caused me quite a bit of chin-scratching on one particular afternoon.